service/src/main/java/bio/terra/cbas/dao/RunDao.java

```diff
@@ -25,7 +25,7 @@ public RunDao(NamedParameterJdbcTemplate jdbcTemplate) {
   public int createRun(Run run) {
     return jdbcTemplate.update(
         "insert into run (id, engine_id, run_set_id, entity_id, submission_timestamp, status)"
-            + " values (:id, :engineId, :runSetId, :entityId, :submissionTimestamp, :status)",
+            + " values (:id, :engineId, :runSetId, :recordId, :submissionTimestamp, :status)",
```

Comment:
I think you also need to update `run` table definition in database by adding a new changeset in `resources/changelog/changesets/`.